src/EditorFeatures/Core.Wpf/SignatureHelp/Controller.Session_ComputeModel.cs

```diff
@@ -174,66 +176,6 @@ private static bool DisplayPartsMatch(SignatureHelpItem i1, SignatureHelpItem i2
 
             private static bool CompareParts(TaggedText p1, TaggedText p2)
                 => p1.ToString() == p2.ToString();
-
-            private static async Task<(ISignatureHelpProvider provider, SignatureHelpItems items)> ComputeItemsAsync(
```

Comment:
Moved into the SignatureHelpService.

Reply:
i'm assuming it didn't change (beyond anything mechanical). if it did, please call that out.

Reply:
I will do a better job of this going forward. Added comments below.